modules/cudacodec/src/video_decoder.cpp

```diff
@@ -136,6 +152,7 @@ void cv::cudacodec::detail::VideoDecoder::create(const FormatInfo& videoFormat)
 #endif
```

Comment:
Need to raise exception for histogram option and old CUDA.

Reply:
Do you mean throw an exception if  `CUDART_VERSION < 9000`.  I'm not sure if disabling histogram output for older versions of the CUDA toolkit is a bit extreme.  As CUDA 9 is over 5 years we could just leave this as it is?